wdl-analysis/src/analyzer.rs

```diff
@@ -409,11 +414,19 @@ where
                 bail!("`{path}` is a file, not a directory", path = path.display());
             }
 
-            for result in WalkDir::new(&path).follow_links(true) {
-                let entry = result.with_context(|| {
-                    format!("failed to read directory `{path}`", path = path.display())
-                })?;
-                if !entry.file_type().is_file()
+            let mut walker = WalkBuilder::new(&path);
+            if let Some(ignore_filename) = config.ignore_filename() {
+                walker.add_custom_ignore_filename(ignore_filename);
+            }
+            let walker = walker.standard_filters(false).parents(true).build();
+
+            for result in walker {
+                let entry = match result {
+                    Ok(entry) => entry,
+                    Err(_) => continue, // ignore any errors parsing the ignore files
+                };
```

Comment:
I don't think this is the behavior we want... an error in an ignore file would make us skip every entry, the result being we'd silently add nothing to the analyzer context. Better to `?` this error out, I'd say (probably with an informative `context`).
```suggestion
                let entry = entry?;
```